nengo_gui/examples/tutorial/02-two-neurons.py

```diff
@@ -19,7 +19,7 @@
     stim = nengo.Node(0)
     ens = nengo.Ensemble(n_neurons=2, dimensions=1, seed=5)
     nengo.Connection(stim, ens)
-    
-    
+
+
     import time
-    slowdown = nengo.Node(lambda t: time.sleep(0.001)) 
\ No newline at end of file
+    slowdown = nengo.Node(lambda t: time.sleep(0.001))
```

Comment:
On the machine I'm working on it happens with both Chrome and Firefox. I also tried on my laptop, the same happens with Firefox, Chrome works slightly better but it's still lagging. I will then remove these lines, but I think it would be useful to mention this elsewhere. Maybe a FAQ or just verbally when introducing tutorials.